config.yml

```diff
@@ -28,6 +28,18 @@ Defaults:
       credentials:
         clientId:     !env TASKCLUSTER_CLIENT_ID
         accessToken:  !env TASKCLUSTER_ACCESS_TOKEN
+    authBaseUrl:      https://auth.taskcluster.net/v1
+    publishMetaData:  false
+  raven:
```

Comment:
Heads up: This is being phased out in favor of having `taskcluster-lib-monitor` set this up automatically. This will totally continue to work for now though, so I think setting that up in a subsequent review might be easiest.